puzzle-utils/words/wordsearch.py

```diff
@@ -79,13 +86,28 @@ def wordsearch_find(grid, words):
     for word in words:
         normalized_words.append(word.lower())
 
-    return wordsearch_grid(grid, wordsearch_find_function, normalized_words)
+    results = wordsearch_grid(grid, wordsearch_find_function, normalized_words)
+
+    missing_words = []
+    for word in words:
```

Comment:
This could be simplified if you created a set out of the results, a set out of the expected words and just found their [symmetric difference](https://docs.python.org/3/library/stdtypes.html#frozenset.symmetric_difference)